cmd/cp.go

```diff
@@ -18,10 +19,45 @@ var CpCommand = &cobra.Command{
 		"Read from stdin: cat /path/to/file | qsctl cp - qs://prefix/stdin",
 		"Write to stdout: qsctl cp qs://prefix/b - > /path/to/file",
 	),
-	Args: cobra.ExactArgs(2),
-	RunE: cpRun,
+	Args:    cobra.ExactArgs(2),
+	RunE:    cpRun,
+	PreRunE: validateCpFlag,
 }
 
 func cpRun(_ *cobra.Command, args []string) (err error) {
 	return action.Copy(args[0], args[1])
 }
+
+func initCpFlag() {
+	CpCommand.PersistentFlags().StringVar(&expectSize,
+		expectSizeFlag,
```

Comment:
Let's use the string directly, and don't need to share flag names between commands.